.github/workflows/testground-publish-edge.yml

```
@@ -0,0 +1,60 @@
+name: Testground Publish (edge)
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    # we release the `edge` binary as an output artifact (see below).
+    # Because these artifacts have a 90 days maximum retention period,
+    # this cron makes sure we always have an `:edge` artifact available.
+    - cron: "0 0 1 * *"
+
+jobs:
+  binaries:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v3
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: "1.16.x"
+        cache: true
+    - name: Build Testground
+      run: go build -ldflags "-X github.com/testground/testground/pkg/version.GitCommit=`git rev-list -1 HEAD`" .
```

Wouldn't that be the same as:
```suggestion
      run: go build -ldflags "-X github.com/testground/testground/pkg/version.GitCommit=${GITHUB_SHA}" .
```

I think it should work for cron triggered workflow run too.